test/local/routes/emails.js

```diff
@@ -802,6 +802,31 @@ describe('/recovery_email', () => {
         () => assert.fail('Should have failed when creating email'),
         err => assert.equal(err.errno, 141, 'cannot add secondary email, newly created primary account'))
     })
+
+    it('fails create email if error while sending verify email', () => {
```

Comment:
nit: '(it) fails create email if error while sending verify email' seems convoluted